catalog/plugin/catalog-plugin-security-audit/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Security Audit Plugin"
+         id="org.codice.ddf.catalog.plugin.security.audit.SecurityAuditPlugin"
+         description="Plugin to audit security attribute changes">
+        <AD name="Security attributes to audit" id="auditAttributes"
+            required="true"
+            type="String"
+            cardinality="1000"
+            description="List of security attributes to audit when modified"
+            default="security.access-groups,security.access-individuals,security.classification,security.classification-system,security.codewords,security.dissemination-controls,security.other-dissemination-controls,security.owner-producer,security.releasability,ext.metadata-originator-classification,ext.metadata-classification,ext.metadata-classification-system,ext.metadata-dissemination-controls,ext.metadata-releasability,ext.resource-originator-classification,ext.resource-classification,ext.resource-classification-system,ext.resource-releasability,ext.resource-dissemination-controls">
```

Comment:
Same thing here for the default. It should just have `security.access-groups,security.access-individuals` in DDF. Alliance can drop a configuration that adds the rest of the attributes to this list.